platform/core/README.md

```diff
@@ -1,8 +1,23 @@
-# Rust implementations of polyPod platform core logic
+# PolyPod platform core
 
-The code is written against Rust 1.58.
+Logic shared across polyPod platforms - ios, android, wasm. 
 
-All you need is _cargo_, then run:
+## Requirements
+- [Rust toolchain](https://www.rust-lang.org/tools/install). Core is built with Rust 1.58.1.
```

Comment:
CMake is now also needed, in order to build flatbuffers, hm? Also Clang or GCC maybe? You could also simply refer to the build requirements of flatbuffers, pointing the user to where they can find out which version of flatbuffers we're going to build.